### PR TITLE
Swap Google forums on Talk page to Meetup Discussion

### PR DESCRIPTION
**Pull request by dmsnell**
We don't use Google anymore for the forums and discussion, but we _do_
want to use the discussion area on Meetup.com. This patch remvoes the
embedded Google talk page and replaces it with an external link to
Meetup.com

cc: @meiqimichelle

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h1 class="h0">Join us</h1>
 			<h2><img src="{{ site.baseurl }}/{{ site.img_dir }}/community.png" alt="Community" class="call-out-img" />Join the discussion</h2>
 			<p>An open conversation online about Tucson, what we're doing now, and what's next.
 			</p>
-			<p><a href="/talk/">Learn and share &raquo;</a></p>
+			<p><a href="http://www.meetup.com/Code-for-Tucson/messages/boards/">Learn and share &raquo;</a></p>
```
Comment (dmsnell) [inline pages/index.html]: good catch @meiqimichelle

**Comment by meiqimichelle** [comment]
I found another reference to the old talk page, which I replaced with the one you used in the nav bar.

**Comment by meiqimichelle** [comment]
hi

**Comment by meiqimichelle** [comment]
hi

**Comment by meiqimichelle** [comment]
:trollface:

**Pull request merged**